packages/fluentui/docs/src/examples/components/Chat/Content/ChatExampleCustomHeader.shorthand.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Avatar,
+  Chat,
+  ChatItemProps,
+  ReactionProps,
+  ShorthandCollection,
+  Text,
+  ReactionGroup,
+} from '@fluentui/react-northstar';
+import * as React from 'react';
+import { EmojiIcon, LikeIcon } from '@fluentui/react-icons-northstar';
+
+const reactions: ShorthandCollection<ReactionProps> = [
+  { key: 'up', icon: <LikeIcon />, content: '1K' },
+  { key: 'smile', icon: <EmojiIcon />, content: 5 },
+];
+
+const items: ShorthandCollection<ChatItemProps> = [
+  {
+    attached: 'top',
+    contentPosition: 'end',
+    message: (
+      <Chat.Message
+        reactionGroup={reactions}
+        content="Hello"
+        header={
+          <>
+            <ReactionGroup items={reactions} />
+            <Text
+              styles={{
+                marginLeft: '5px',
```

Comment:
Can we avoid style overrides and use `Flex` with a `gap`:

```
<Flex gap={3}>
  <ReactionGroup items={reactions} />
  <Text />
  <Text />
</Flex>
```

Reply:
if we do that we end up with a html like:

```html
<div> /* HEADER */
    <div> /* FLEX */

```